docker/build.run.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+check_vols_src() {
+  if [ ! -d /vols/src ]; then
+    echo "No /vols/src with code"
+    exit 1
+  fi
+}
+check_vols_out() {
+  if [ ! -d /vols/out ]; then
+    echo "No /vols/out for output!"
+    exit 1
+  fi
+}
+
+function sync {
+  check_vols_src
+  {
+    for f in bin/*; do
+      echo "- ${f}"
```

Comment:
Avoiding bringing useless stuff into the tar and not overwriting the dependencies installed during the Docker build